UI.py

```diff
@@ -208,6 +208,50 @@ def CreateIsMemberWorkingOptions():
 
     return member_options
 
+def CreateGetLeavesAcrossRangeOptions():
+    member_options = [
+        create_option(
+            name = "startdate",
+            description = "start date (inclusive) in DD/MM/YYYY format",
+            option_type = SlashCommandOptionType.STRING,
+            required = True
+        ),
+        create_option(
+            name = "enddate",
+            description = "end date (inclusive) in DD/MM/YYYY format",
+            option_type = SlashCommandOptionType.STRING,
+            required = True
+        ),
+        create_option(
+            name = "discorduser",
+            description = "discord user",
```

Comment:
A note that leaving this empty will query for everyone would be nice

Reply:
True, Done